backend/app/services/config_service.py

```
@@ -0,0 +1,168 @@
+        cursor.execute(
+            """
+            INSERT INTO field_mappings (domain, field_name, cv_path, confidence)
+            VALUES (?, ?, ?, ?)
+        """,
+            (domain, field_name, cv_path, confidence),
+        )
+
+        db_channel.commit()
+        db_channel.close()
+
+    def get_learned_mappings(self, domain: str) -> List[Dict[str, Any]]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)